<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filter_lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sqlUser_lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="refresh_commandLinkButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="finished_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Finished</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Sample Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Temperature</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Bias</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>Sample1</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string>First</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string>Second</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Sample:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="selectedSample_label">
            <property name="text">
             <string>No Sample Selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Background:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="selectedBackground_label">
            <property name="text">
             <string>No Background Selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
